.github/workflows/codeql.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "develop" ]
+  schedule:
+    - cron: '21 11 * * 0'
```

Comment:
> 21 11 * * 0

That cron expression translates to "11:21am every Sunday". Any reason for that?

Alternatives would be:
- 10am every sunday: `0 10 * * 0`
- 10am every day: `0 10 * * *`
- 10am every week days: `0 10 * * 1-5`